resource_access/item_instance.gd

```diff
@@ -86,7 +86,20 @@ func get_item_flavor_text() -> String:
 ## after you call this you must add the instanced scenes to the scene tree
 func spawn_item() -> void:
 	if _item_info.has_stacks:
-		var stack:int = randi_range(1, _item_info.max_stacks)
+		print('HAS STACKS')
```

Comment:
I appreciate the enthusiasm but we can probably cut this or make it a debug print instead